### PR TITLE
feat/register with year and gender

### PR DESCRIPTION
**Pull request by new5558**

### CODE REVIEW

File: src/commons/utils/validForm.ts [hunk 1]
```diff
@@ -42,3 +42,10 @@ export function validateProvince(province: string) {
   const expression = /[^0-9]/;
   return expression.test(province);
 }
+
+export function validateYear(year: number | null) {
+  if (!year) {
+    return true;
```
Comment (poom-kitti) [inline src/commons/utils/validForm.ts:48]: Why true if do not have year?

File: src/modules/signup/modal-additional-info/ModalAdditionalInfo.vue [hunk 2]
```diff
@@ -45,6 +45,35 @@
             </div>
           </div>
         </section>
+        <section class="mb-2">
+          <div class="flex space-between">
+            <div class="flex flex-col mr-2 w-full">
+              <label for="year" class="mb-0.25">Gender</label>
```
Comment (poom-kitti) [inline src/modules/signup/modal-additional-info/ModalAdditionalInfo.vue]: Why for="year" here?

**Review by poom-kitti — APPROVED** [review]
LGTM after fix

File: src/commons/utils/validForm.ts (hunk 1 above)
Comment (new5558) [reply src/commons/utils/validForm.ts:48]: Because this validate is for when input error not when there is not input

**Pull request merged**